webapp/src/index.tsx

```diff
@@ -7,16 +7,25 @@ import manifest from './manifest';
 // eslint-disable-next-line import/no-unresolved
 import {PluginRegistry} from './types/mattermost-webapp';
 
+const globalStats = {
+    Start: Date.now(),
+    NumDispatches: 0,
```

Comment:
This is very coarse -- a better solution might be to track in buckets of time, so we can correlate perceived performance with the number of dispatches observed near that unit of time. But this might be good enough as a baseline.

Reply:
We could also add something to count which actions were dispatched the most too